### DISCUSSION TITLE
Add default value to documentation

### DISCUSSION THREAD

**Discussion by Florian-crg** [biomejs/biome#3599] [Ideas]
If you look to the biome documentation at https://biomejs.dev/reference/cli/#biome-ci for example, there is not always the default value. 

> --vcs-enabled=<true|false> — Whether Biome should integrate itself with the VCS client

Would be nice to know the default value of all of the properties. Like for the one below for instance.
> --attribute-position=<multiline|auto> — The attribute position style in HTMLish languages. By default auto.

**Comment by ematipico** [comment]
There aren't default values because the default values that Biome uses don't come from the CLI, but from Biome itself. If you don't pass a flag/option via CLI, Biome uses a set inside its core.

It's done in this way because Biome is meant to work with different clients, and the CLI is only one of them.

**Reply by Florian-crg** [reply]
> Biome uses a set inside its core.

To double check I properly understood, in my case is it the biome.json that set the default values?
If not what are the default values inside its core?

I'm not sure to understand how it can have no default value. It is either `undefined`, `true` or `false` for a boolean 🤔

**Reply by ematipico** [reply]
If you don't provide any configuration or options via CLI, Biome uses its defaults. So maybe you're asking which ones are the defaults.

There are different ways to know that: 
- The website
	-  Formatter: https://biomejs.dev/formatter/#options
	- Linter: https://biomejs.dev/linter/rules/#recommended-rules
- With the `rage` command: `biome rage --formatter` or `biome rage --linter`

**Reply by Florian-crg** [reply]
Ok understood, thanks for letting me know.

If I had the question, probably that others will too. 

Maybe that we could have an indication at the field levels, using an asterix, a link or else  🤷 

--vcs-enabled=<true|false> — Whether Biome should integrate itself with the VCS client. By default _explanation_link_.
